tool/grind.dart

```diff
@@ -44,6 +46,17 @@ void main(List<String> args) {
   pkg.standaloneName.value = "dart-sass";
   pkg.githubUser.fn = () => Platform.environment["GH_USER"];
   pkg.githubPassword.fn = () => Platform.environment["GH_TOKEN"];
+  // Limit the methods exposed as ESM exports to those that don't require
+  // filesystem access
```

Comment:
Is this still accurate?

Reply:
Removed in https://github.com/oddbird/dart-sass/commit/1937affd6fa9fc3965c72222f323f56fb53352d7